client/starwhale/api/_impl/job.py

```diff
@@ -40,6 +41,8 @@ def __init__(
         expose: int = 0,
         virtual: bool = False,
         require_dataset: bool = False,
+        parameters_sig: t.List[t.Dict[str, object]] = [],
```

Comment:
`[]` should not be as the default value.

Reply:
done